components/kyma-environment-broker/cmd/broker/main.go

```diff
@@ -37,10 +45,10 @@ type Config struct {
 	Host string `envconfig:"optional"`
 	Port string `envconfig:"default=8080"`
 
-	Provisioning       input.Config
-	Director           director.Config
-	Database           storage.Config
-	ManagementPlaneURL string
```

Comment:
do not remove it

Reply:
It has been discussed. I have restored this filed